src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -823,11 +823,13 @@ private static void createAndCheckIsDirectory(Path dir,
      * names in the same manner as the {@link
      * java.io.File#createTempFile(String,String,File)} method.
      *
-     * <p> The file may be opened using the {@link
+     * <p> As with the {@code File.createTempFile} methods, this method is only
+     * part of a temporary-file facility. Where the resulting file is used
+     * as a <em>work file</em> solely by a single instance of the Java virtual
```

Comment:
I think the insertion of "solely by a single instance of the Java virtual machine" will create questions and we'll end up re-writing the paragraph again. I think I'd prefer to just restore the text and then start again with the questions that lead to the change in the first place.